cluster_config.tf

```diff
@@ -0,0 +1,106 @@
+resource "aws_emr_security_configuration" "ebs_emrfs_em" {
+  name          = "dataworks_aws_collections_rehydration_ebs_emrfs"
+  configuration = jsonencode(local.ebs_emrfs_em)
+}
+
+#TODO remove this
+output "security_configuration" {
```

Comment:
Is this your "Todo"?

Reply:
I think I shell remove it if it is not breaking anything

Reply:
removed